benchmark/benchmark_config_dispatch.cpp

```diff
@@ -91,20 +91,19 @@ int main(int argc, char** argv)
 
     // Parse argv
     benchmark::Initialize(&argc, argv);
-    const size_t size   = parser.get<size_t>("size");
     const int    trials = parser.get<int>("trials");
     bench_naming::set_format(parser.get<std::string>("name_format"));
 
-
     // HIP
 
     std::vector<benchmark::internal::Benchmark*> benchmarks{
         CREATE_BENCHMARK(default_stream, stream_kind::default_stream),
         CREATE_BENCHMARK(per_thread_stream, stream_kind::per_thread_stream),
         CREATE_BENCHMARK(explicit_stream, stream_kind::explicit_stream),
-        CREATE_BENCHMARK(async_stream, stream_kind::async_stream)
-    };
-
+        CREATE_BENCHMARK(async_stream, stream_kind::async_stream),
+        benchmark::RegisterBenchmark(
+            bench_naming::format_name("{lvl:na,algo:empty_kernel,cfg:default_config}"),
```

Comment:
This does not compile:

/tmp/grpc/rocPRIM/benchmark/benchmark_config_dispatch.cpp:104:9: error: no matching function for call to 'RegisterBenchmark'
  104 |         benchmark::RegisterBenchmark(
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/include/benchmark/benchmark.h:1182:29: note: candidate function not viable: no known conversion from 'std::string' (aka 'basic_string<char>') to 'const char *' for 1st argument
 1182 | inline internal::Benchmark* RegisterBenchmark(const char* name,
      |                             ^                 ~~~~~~~~~~~~~~~~
/usr/local/include/benchmark/benchmark.h:1190:22: note: candidate function template not viable: no known conversion from 'std::string' (aka 'basic_string<char>') to 'const char *' for 1st argument
 1190 | internal::Benchmark* RegisterBenchmark(const char* name, Lambda&& fn) {
      |                      ^                 ~~~~~~~~~~~~~~~~
/usr/local/include/benchmark/benchmark.h:1201:22: note: candidate function template not viable: no known conversion from 'std::string' (aka 'basic_string<char>') to 'const char *' for 1st argument
 1201 | internal::Benchmark* RegisterBenchmark(const char* name, Lambda&& fn,

Adding `.c_str()` fixes the compile error.

Reply:
Should be fixed now!